Tasks/VsTest/vstest.ts

```diff
@@ -108,8 +108,13 @@ function invokeVSTest(testResultsDirectory: string): Q.Promise<number> {
                     setRunInParallellIfApplicable(vsVersion);
                     setupRunSettingsFileForParallel(runInParallel, overriddenSettingsFile)
                         .then(function(parallelRunSettingsFile) {
-                            var vsCommon = tl.getVariable("VS" + vsVersion + "0COMNTools");
-                            var vstestLocation = path.join(vsCommon, "..\\IDE\\CommonExtensions\\Microsoft\\TestWindow\\vstest.console.exe");
+                            var vstestPath = tl.getVariable("VSTest_" + vsVersion.toFixed(1));
+                            if (!vstestPath) {
+                                tl.warning(tl.loc('VstestNotFound', vsVersion));
+                                defer.resolve(1);
```

Comment:
then the warning shuld be made error?

---
In reply to: [67283921](https://github.com/Microsoft/vsts-tasks/pull/1886#discussion_r67283921) [](ancestors = 67283921)